src/main/resources/templates/translatemessage.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Translated GilfMessage</title>
```

Comment:
Cursed or blessed title? 😄

Reply:
Hö hö ^^ Ofc its Gilfoyle from Silicon Valley --> [Clean code challenge](https://youtu.be/qNj_sI6qndw?feature=shared&t=31)

![image](https://github.com/Win-ther/SpringbootGroupProject/assets/143100902/95014b46-1672-44dd-88ea-c0dd0e5dc17f)

Reply:
Blessed.